Userland/Libraries/LibWeb/HTML/EventLoop/EventLoop.h

```diff
@@ -72,6 +77,10 @@ class EventLoop {
     // https://html.spec.whatwg.org/multipage/webappapis.html#currently-running-task
     Task* m_currently_running_task { nullptr };
 
+    // https://html.spec.whatwg.org/multipage/webappapis.html#last-render-opportunity-time
+    double m_last_render_opportunity_time { 0 };
+    double m_last_idle_period_start_time { 0 };
```

Comment:
Please add a separate link for this: https://html.spec.whatwg.org/multipage/webappapis.html#last-idle-period-start-time